starter/script.js

```diff
@@ -1 +1,39 @@
-// put code here!
+var userInput;
+var suchen = function(message){
+  userInput=prompt(message);
+  if (userInput){
+    return userInput;
+  } else {
+    suchen('TRY AGAIN: ' + message);
+  }
+};
+
+var game = function  () {
+  suchen('Hey! What is your name??');
+  suchen('Hi there, ' + userInput + '. Wanna play a game?? (y|n)');
+  if (userInput=='y'){
```

Comment:
try to avoid global variables in the future and instead rely on the return value of `suchen()` here